SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security Policy
+
+## Security
+
+At Autodesk, we know that the security of your data is critical to your studio’s
+operation.
+As the industry shifts to the cloud, ShotGrid knows that security and service
+models are more important than ever.
+
+The confidentiality, integrity, and availability of your content is at the top
+of our priority list.
+Not only do we have a team of ShotGrid engineers dedicated to platform security
+and performance, we are also backed by Autodesk’s security team, also invests
```

Comment:
we are also backed by Autodesk’s security team, also invests
might be better with:
we are also backed by Autodesk’s security team that also invests